src/slates.nim

```diff
@@ -0,0 +1,61 @@
+import flippy, snappy, streams, strformat
+
+type Slate* = object
+  mipmaps*: seq[Image]
+
+func width*(slate: Slate): int =
+  slate.mipmaps[0].width
+
+func height*(slate: Slate): int =
+  slate.mipmaps[0].height
+
+proc save*(slate: Slate, filePath: string) =
```

Comment:
Can you add a simple slate load/save test?

Reply:
Sure, done.